app/main/routes.py

```diff
@@ -66,7 +66,10 @@ def upload():
             current_app.logger.info(f"Upload successful: {metadata}")
             return render_template("success.html", file_name=excel_file.filename)
         elif pre_errors:
-            current_app.logger.info("Pre-validation errors found during upload")
+            if Config.ENABLE_VALIDATION_LOGGING:
+                current_app.logger.info(f"Pre-validation error(s) found during upload: {pre_errors}")
+            else:
+                current_app.logger.info("Pre-validation error(s) found during upload")
```

Comment:
We can combine this if/else into a single line by doing:

```
current_app.logger.info("Pre-validation error" + f": {pre_errors}" if Config.ENABLE_VALIDATION_LOGGING else "")
```

Reply:
Having tested this when ENABLE_VALIDATION_LOGGING is set to false, nothing gets printed out as the if statement doesn't distinguish between the concatenated strings (its treated like a single string which doesn't get logged if set to false.)

I could still make this a one-liner by adding the sentence inside else "", however this would need to be split up because of character limits anyway. I find using plain if/else statements like this is the most readable despite taking up more lines, which is why I usually implement them over more intricate looking solutions - but I'm happy to write the code in a way the majority agrees with. Let me know your thoughts

Reply:
Fair enough - both perfectly valid options. I think I generally find additional nesting less readable and would argue the logic is quite simple and clear in this case.

I would also say as a general rule I think we've tended to opt with including simple logical statements into single lines throughout the codebase so I think it fits in consistency wise. 

Happy to go with the majority too.

@Connor-OS  any thoughts?

Reply:
I thinkn it's quite a pythonic way to write if statments which can sometimes make it harder to pass than if else blocks especially if you're packing a lot of logic into a single line. In this case it makes sense to me to use a one line statment but I wouldn't fault anyone for opting to split it out and both are very readable 👍